packages/apify/src/enqueue_links/click_elements.ts

```diff
@@ -201,14 +223,8 @@ export async function clickElementsAndInterceptNavigationRequests(options): Prom
     return serializedRequests.map((r) => JSON.parse(r));
 }
 
-/**
- * @param {Page} page
- * @param {Set<*>} requests
- * @return {Function}
- * @ignore
```

Comment:
keep the @ignore comment, that is for typedoc so it is not part of the API documentation